src/gatsby-ssr.js

```diff
@@ -1,23 +1,71 @@
 const React = require('react')
 
+const defaultDomain = 'cdn.simpleanalytics.io'
+
 exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
-  const domain = pluginOptions.domain || 'cdn.simpleanalytics.io'
-  const scriptName = domain === 'cdn.simpleanalytics.io' ? 'hello.js' : 'app.js'
+  const domain = pluginOptions.domain || defaultDomain
+  let scriptName
+  if (domain === defaultDomain) {
+    scriptName = 'hello.js'
+  } else {
+    scriptName = 'app.js'
+  }
   const scriptUrl = `https://${domain}/${scriptName}`
 
   const options = {
-    key: 'plugin-simpleanalytics',
     src: scriptUrl,
-    async: true,
-    defer: true
+    async: true
   }
 
   if (pluginOptions.metomic) {
     options.type = 'text/x-metomic'
-    options['data-micropolicy'] = pluginOptions.metomic
+    options.metomic = pluginOptions.metomic
+  }
+  if (pluginOptions.events) {
+    options.eventsGlobal = pluginOptions.eventsGlobal
   }
 
   setHeadComponents([
-    React.createElement('script', options)
+    React.createElement('script', {
+      key: 'plugin-simpleanalytics',
+      dangerouslySetInnerHTML: {
+        __html: loadScript(domain, options)
+      }
+    })
   ])
 }
+
+// this script runs when the page loads, it does the following;
+// 1. Checks the doNotTrack option, and if set then it aborts and does nothing
+// 2. Creates a script tag for the Simple Analytics script
+// 3. Sets all the options properties on the script tag
+// 4. Inserts the script tag before the first other script in the document
+const loadScript = (domain, options) => {
+  return `!(function(s, i, m, p, l, e) {
+    const dnt = s.doNotTrack || m.doNotTrack || m.msDoNotTrack;
```

Comment:
`const` is not supported in every browser, or will this be compiled as well?

Reply:
Yeah it's compiled with babel.